docs/guides/pdf-non-ascii-languages.rst

```diff
@@ -0,0 +1,59 @@
+Build PDF format for non-ASCII languages
+========================================
+
+
+.. warning::
+
+   To be able to follow this guide and build PDF with this method,
+   you need to ask to enable ``USE_PDF_LATEXMK`` :doc:`feature flag </guides/feature-flags>` in your project.
+   Please, `open an issue`_ in our repository asking for this and wait for one of the core team to enable it.
+
+.. _open an issue: https://github.com/rtfd/readthedocs.org/issues/new
+
+
+Sphinx comes with support for different `LaTeX engines`_ that support non-ASCII languages,
+like Japanese or Chinese, for example.
+By default Sphinx uses ``pdflatex``,
+which does not have good support for Unicode characters and may make the PDF builder to fail on these languages.
+
+.. _LaTeX engines: http://www.sphinx-doc.org/en/master/usage/configuration.html#confval-latex_engine
+
+In case you want to build your documentation in PDF format you need to configure Sphinx properly,
+so Read the Docs can execute the proper commands depending on these settings.
+There are `several settings that can be defined`_ (all the ones starting with ``latex_``),
+to modify Sphinx and Read the Docs behavior to make your documentation to build properly.
+
+A good first try would be to use ``latex_engine = 'xelatex'`` only,
+if your docs are not written in Chinese or Japanese.
+This is because your build could be failing just because of an Unicode error
```

Comment:
Although this might be obvious to experienced folks, it is not clear that settings should be added to the project's `conf.py` file. I'd suggest inserting a sentence:

```
Edit your project's documentation configuration file ``conf.py``.
A good first try....
```